api.planx.uk/send/exportZip.ts

```diff
@@ -65,7 +65,8 @@ export async function buildSubmissionExportZip({
   }
 
   // generate csv data
-  const { responses, redactedResponses } = await $api.export.csvData(sessionId);
+  const responses = await $api.export.csvData(sessionId);
+  const redactedResponses = await $api.export.csvDataRedacted(sessionId);
```

Comment:
This is the only place where both redacted and un-redacted payloads were used. This means that these two calls will now be _slightly_ less efficient as the `ExportClient` is making additional requests to the DB (e.g. to fetch session data).

I think this is a fair compromise as... 
- We only need both in one place (vs. all other methods touched in this PR)
- Fetching from the DB is the quick / "cheap" part of the function that's being called under the hood - and our GraphQL client (or DB / Hasura) is very likely doing some caching of these requests for us anyway.